e2e_test/batch/describe_fragments.slt

```diff
@@ -95,10 +95,9 @@ DESCRIBE FRAGMENTS describe_plan_test.src;
 ----
 db error: ERROR: Failed to run the query
 
-Caused by:
-  Not supported: non shared source has no fragments to describe
-HINT: Use `DESCRIBE` instead.
-
+Caused by these errors (recent errors listed first):
+  1: Catalog error
+  2: materialized view, index, table or sink not found: describe_plan_test.src
```

Comment:
I think previous error msgs are more user-friendly. Can we keep that?

Reply:
I elaborated in my PR description for this part.

Reply:
I can add more description, tell users not to describe fragments their `source` and `view` I think that's probably the most likely.